src/tokens/BlessCurseManager.ttslua

```diff
@@ -156,289 +117,359 @@ function initializeState()
         end
     end
 
-    mode = "Bless"
-    print("Bless Tokens " .. getTokenCount())
-    mode = "Curse"
-    print("Curse Tokens " .. getTokenCount())
+    print("Bless Tokens " .. getTokenCount("Bless"))
```

Comment:
I'll defer to Tika but I think we can remove this.  IMHO it's spammy and not really necessary on load.

Reply:
What about only printing this if one of the counts is nonzero? I quite like the quick information when loading games